scrap.js

```diff
@@ -58,9 +60,18 @@ const getChannelVideos = async (channelId, maxPage)  => {
   const url = `https://m.youtube.com/channel/${channelId}/videos`
   const videos = parseVideos(await getURLContents(url, maxPage));
   console.log('video list', videos, `Total ${videos.length} videos`);
+
   return videos;
 }
 
+const getVideoDetails = (videos = []) => {
+  return videos.map(async (videoId) => {
```

Comment:
let's construct it inside this function `const youtube = new Youtube();`

Reply:
not sure about this ! I have moved `const youtube = new Youtube();` inside of the function `getVideoDetails` as per comment above

Reply:
yes, i meant that!